### PR TITLE
Bump Nodejs to v16

### PR DESCRIPTION
**Pull request by WatanabeToshimitsu**
## What?
- Bump Nodejs to v16
- Add config for volta 
- Add config to check version of npm and nodejs strictly when install npm packages

## Why?
- Nodejs v16 will be LTS soon

### CODE REVIEW

**Comment by hdl-service** [comment]
[APPROVALNOTIFIER] This PR is **APPROVED**

This pull-request has been approved by: *<a href="https://github.com/dataware-tools/app-template-nextjs/pull/88#" title="Author self-approved">WatanabeToshimitsu</a>*

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=dataware-tools%2Fapp-template-nextjs).

The pull request process is described [here](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process)

<details >
Needs approval from an approver in each of these files:

- ~~[OWNERS](https://github.com/dataware-tools/app-template-nextjs/blob/master/OWNERS)~~ [WatanabeToshimitsu]

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

File: .npmrc [hunk 1]
```diff
@@ -1 +1,2 @@
 @dataware-tools:registry=https://npm.pkg.github.com
+engine-strinct=true
```
Comment (yusukefs) [inline .npmrc:2]: `strinct` となってるんですが多分 `strict` の間違いですかね？

Comment (WatanabeToshimitsu) [reply .npmrc]: タイポマンだ....
修正しました！

**Comment by hdl-service** [comment]
[APPROVALNOTIFIER] This PR is **APPROVED**

This pull-request has been approved by: *<a href="https://github.com/dataware-tools/app-template-nextjs/pull/88#" title="Author self-approved">WatanabeToshimitsu</a>*, *<a href="https://github.com/dataware-tools/app-template-nextjs/pull/88#pullrequestreview-778104686" title="Approved">yusukefs</a>*

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=dataware-tools%2Fapp-template-nextjs).

The pull request process is described [here](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process)

<details >
Needs approval from an approver in each of these files:

- ~~[OWNERS](https://github.com/dataware-tools/app-template-nextjs/blob/master/OWNERS)~~ [WatanabeToshimitsu,yusukefs]

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Pull request merged**